onet-parse/src/create-sql.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This file creates the sql statements and inserts into the sql file
+ */
+import fs from 'fs';
+import { ONETJobDetailResponse } from './types';
+
+/**
+ * Creates the following sql files:
+ * - onet-job-profiles.sql 
+ * - onet-job-info.sql
+ * - recommended-job-list.sql
+ */
+export const createSql = (data: Array<ONETJobDetailResponse>) => {
+    let onetJobInfoInserts = '';
+    let onetJobProfilesInserts = '';
+    let recomnededJobListInserts = '';
+    // TODO: finalize create table statements so the insert statement makes sense
```

Comment:
yeah, we should try to settle on our tables soon. It's gonna affect our side of parsing too.